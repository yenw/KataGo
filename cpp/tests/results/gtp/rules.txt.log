: GTP Engine starting...
: KataGo v1.3.2
: Using 1 CPU thread(s) for search
: nnRandSeed0 = forTesting
: After dedups: nnModelFile0 = models/g170-b6c96-s175395328-d26788732.bin.gz useFP16 false useNHWC false
: Cuda backend: Found GPU Tesla V100-SXM2-16GB memory 16945512448 compute capability major 7 minor 0
: Cuda backend: Model version 8 useFP16 = false useNHWC = false
: Cuda backend: Model name: g170-b6c96-s175395328-d26788732
: Loaded neural net with nnXLen 19 nnYLen 19
: Loaded model models/g170-b6c96-s175395328-d26788732.bin.gz
: Model name: g170-b6c96-s175395328-d26788732
: GTP ready, beginning main protocol loop
: Controller: kata-get-rules
: = {"hasButton":false,"ko":"POSITIONAL","scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
: Controller: kata-set-rules japanese
: = 
: Controller: kata-get-rules
: = {"hasButton":false,"ko":"SIMPLE","scoring":"TERRITORY","suicide":false,"tax":"SEKI","whiteHandicapBonus":"0"}
: Controller: kata-set-rules chinese
: = 
: Controller: kata-get-rules
: = {"hasButton":false,"ko":"SIMPLE","scoring":"AREA","suicide":false,"tax":"NONE","whiteHandicapBonus":"N"}
: Controller: kgs-rules chinese
: = 
: Controller: kata-get-rules
: = {"hasButton":false,"ko":"POSITIONAL","scoring":"AREA","suicide":false,"tax":"NONE","whiteHandicapBonus":"N"}
: Controller: kgs-rules new_zealand
: = 
: Controller: kata-get-rules
: = {"hasButton":false,"ko":"SITUATIONAL","scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
: All cleaned up, quitting
