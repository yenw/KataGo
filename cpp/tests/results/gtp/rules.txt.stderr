KataGo v1.3.2
Loaded model models/g170-b6c96-s175395328-d26788732.bin.gz
Model name: g170-b6c96-s175395328-d26788732
GTP ready, beginning main protocol loop
